pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
```

Comment:
Yes, but confirmation of what? Just the transfer? That sounds weird, i.e. as if I'd say "copy files x y and z to foo", and the tool asked me for each of those "do you really want to copy x/y/z?".

I could understand for overwriting/deleting, and for commands using glob patterns, which could potentially catch more stuff than intended, but if it's indeed applied as a general rule, it strikes me as odd. I'm just trying to figure out which one is the case.